src/main/scala/io/iohk/ethereum/jsonrpc/FilterManager.scala

```diff
@@ -124,13 +132,19 @@ class FilterManager(
       }
     }
 
+    val bestBlockNumber = appStateStorage.getBestBlockNumber()
+
     val fromBlockNumber =
-      startingBlockNumber.getOrElse(resolveBlockNumber(filter.fromBlock.getOrElse(BlockParam.Latest), appStateStorage))
+      startingBlockNumber.getOrElse(resolveBlockNumber(filter.fromBlock.getOrElse(BlockParam.Latest), bestBlockNumber))
```

Comment:
Perhaps this was address before, but should `Latest` be the default? I would expect that not providing `fromBlock` param would result in a block number in the past (`Earliest` ?). Otherwise, when `fromBlock` is not provided`, it only makes sense to have `toBlock == Latest`.

Is this how other clients behave?